docs/configuration.rst

```diff
@@ -25,136 +25,171 @@ Mapcrafter render something (let's call it ``render.conf``):
     rotations = top-left top-right bottom-right bottom-left
-    texture_dir = textures/special_textures
+    block_dir = data/special_blocks
     texture_size = 16
 
-Here we have some more worlds and maps defined. We have a "normal" world which
-is rendered with the day, night, cave render mode, and also with the top view
-and a lower texture size as overview map. Also we have a "creative" world which
-is rendered with a special texture pack, higher texture size and all available
-world rotations with the day and night render mode (super fancy!).
-
-As you can see there is a new section ``global:map``. This section is used to
-set default values for all map sections. Because of this in this example every
-map has the world ``world``, the 3D isometric render view, the daylight render
-mode, the world rotations top-left and top-right and the 12px texture size as
-default. Of course you can overwrite these settings in every map section.  There
-is also a global section ``global:world`` for worlds, but at the moment there is
-only one configuration option for worlds (``input_dir``), so it doesn't make
-much sense setting a default value here.
-
-Furthermore every map has as option ``name`` a name which is used in the web
-interface of the output HTML-File. This can be anything suitable to identify
-this map. In contrast to that the world and map names in the sections are used
-for internal representation and therefore should be unique and contain only
-alphanumeric chars and underscores.
-
-When you have now your configuration file you can render your worlds with (see
-:ref:`command_line_options` for more options and usage)::
-
-    mapcrafter -c render.conf
+You can find other `example maps <https://github.com/mapcrafter/mapcrafter/wiki/Example-maps>`_ 
+on the Github Wiki. Other configuration examples may be available there.
 
 There are tons of other options to customize your rendered maps. Before a
-reference of all available options, here is a quick overview of interesting
-things you can do:
+reference of fundamentals and all available options, here is a quick overview 
+of interesting things you can do:
 
 * Default view / zoom level / rotation in web interface
 * World cropping (only render specific parts of your world)
 * Block mask (skip rendering / render only specific types blocks)
 * Different render views, render modes, overlays
-* Use custom texture packs, texture sizes, apply a blur effect to textures
+* Use custom texture packs, or texture sizes
 * Custom tile widths
 * Different image formats
 * Custom lighting intensity
 
 
+Configuration File Fundamentals
+===============================
+
+Configuration files consist of several sections (e.g. ``[section]``) and 
+*key+value* pairs (e.g. ``key = value``). Sections are named after a colon
+(eg: ``[section:name]``). Lines starting with a hash (``#``) are ignored and can
+be used as comments or notes.
+
+There are four sections which can configure Mapcrafter output:
```

Comment:
Maybe `section types` instead of `sections` ?

Reply:
Sure.